demo/dvc.lock

```diff
@@ -17,18 +17,18 @@ stages:
         weight_decay: 0
     outs:
     - path: misclassified.jpg
-      md5: 5edd7dd13144d95ef72e270570d9d338
```

Comment:
Were the changes to `demo` unintentional?

Reply:
Thanks, I went too fast on that one, I didn't even played with the demo project so I didn't think of checking for a simple change 🤦🏼‍♀️